mindjunkies/courses/management/commands/populate_category.py

```
@@ -0,0 +1,48 @@
+            {"name": "Translation Studies", "description": "Develop skills in linguistic translation and interpretation."},
+            {"name": "Health & Fitness", "description": "Explore wellness, nutrition, and exercise."},
+            {"name": "Yoga & Meditation", "description": "Practice mindfulness and physical wellness techniques."}
+        ]
+        for cat in categories:
+            category, created = CourseCategory.objects.get_or_create(
+                name=cat["name"],
+                defaults={"description": cat["description"]}
+            )
+            self.stdout.write(f"{'Created' if created else 'Updated'} category: {category.name}")
+
+        self.stdout.write(self.style.SUCCESS("Course category data population completed successfully!"))
```

Ruff (0.8.2)

No newline at end of file

Add trailing newline

(W292)